jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -968,6 +968,17 @@ static void completeInitialization() {
 	initSystemClassLoader = true;
 }
 
+/*[IF Sidecar18-SE-OpenJ9|Sidecar19-SE]*/
+//Returns incoming class's classloader without going through security checking
+static ClassLoader getClassLoader(Class<?> clz) {
+	if (null != clz) {
+		return clz.getClassLoader0();
+	} else {
```

Comment:
< grumbles about else blocks after return unconditional returns >

(I don't expect this to be changed here as it was pre-existing code)